src/css/styles.scss

```diff
@@ -169,3 +169,22 @@ form input {
   justify-content: space-around;
   overflow: scroll;
 }
+
+//DESTINATION BOOKING PAGE
```

Comment:
Awesome to have notes like that clarifying styles!

Reply:
Yeah, this also helps guide me when I'm thinking about where I need to put the rules for a certain style.